canonical-web-and-design/maas-ui
# Revert "feat(ui): add TLS certificate to API authentication page (#3819)"

## Done

- This reverts commit afb62b9cadb24efa6f0267f53e0a4b4f6c7f8c96.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to user preferences and see that the "API authentication" tab is reverted back to "API keys"
- Check that the API keys tab does not show any info on the TLS certificate fingerprint

## Fixes

Fixes #3842
